README.md

```diff
@@ -1,69 +1,54 @@
 # ModuleTestingEnvironment
 
-A test helper to instantiate a full headless TerasologyEngine instance
+A test helper to instantiate a full headless TerasologyEngine instance in JUnit tests.
 
 ## Usage
 
-There are two options how to use the **Module Testing Environment** for writing module tests.
-
-The first variant is to write a test class that `extends ModuleTestingEnvironment`. 
-With this setup, a new engine will be spun up for each method annotated with `@Test` automatically.
-
-For more control and possible reuse of engine instances for multiple tests the module testing environment can be used explicitly. 
-This requires manual setup and tear down, but allows to run multiple tests against the same engine instance.
-
 For complete docs please see the
-[documentation on Github Pages](https://terasology.github.io/ModuleTestingEnvironment/org/terasology/moduletestingenvironment/ModuleTestingEnvironment.html)
+[documentation on Github Pages](https://terasology.github.io/ModuleTestingEnvironment/org/terasology/moduletestingenvironment/MTEExtension.html)
 
 For more examples see
 [the test suite](https://github.com/terasology/ModuleTestingEnvironment/tree/master/src/test/java/org/terasology/moduletestingenvironment)
 
 Here's an example taken from the test suite:
 
 ```java
-public class MyModuleEngineTest extends ModuleTestingEnvironment {
-    protected Logger logger = Logger.getLogger(MyModuleEngineTest.class.getName());
-    protected EntityManager entityManager;
-
-    @Override
-    public Set<String> getDependencies() {
-        return Sets.newHashSet("MyModule");
-    }
-
-    @Override
-    public String getWorldGeneratorUri() {
-        return "mymodule:mymodulesworldgenerator";
```

Comment:
This had an example how to set the world generator - what would be the equivalent now?